### DISCUSSION TITLE
PubPub Books pdf generation and sequential reading flow

### DISCUSSION THREAD

**Discussion by AnniHuch** [knowledgefutures/pubpub#3189] [Support]
Dear pubpub Team,

I wanted to request two things regarding the pubpub Book format. We just transferred our lecture notes to a book but we miss two features. Can you help me with the settings?

1. How to export a whole book as pdf?
2. Is there any option to add some "go to previous chapter" or "go to next chapter" buttons in the end of each pub in order to provide a more sequential reading flow to the reader?

Here is the Link to our book: https://theorinf.pubpub.org/vorlesungsskript

Best regards and thanks in advance
Annika

**Comment by eastofesten** [comment]
Hi Annika - thanks for reaching out! 

1. PubPub Legacy does not have a way to export a whole book as a PDF from the front-end.
2. PubPub Legacy does not have a way to add additional . We normally recommend using the collection drop-down at the top of the page (see image below) to move between chapters. 

<img width="1440" alt="Screen Shot of theorinf.pubpub.org to display collection drop-down menu" src="https://github.com/user-attachments/assets/8971bb80-7f64-47c7-8cc6-f09f34394dd9">
